.appveyor.yml

```diff
@@ -7,7 +7,7 @@ build:
   verbosity: detailed
 
 build_script:
-  - gradlew.bat --no-daemon --stacktrace --warning-mode=all checkstyle check build
+  - gradlew.bat clean --no-daemon --stacktrace --warning-mode=all checkstyle check build
```

Comment:
`clean` could come after all options.

Reply:
Thanks! This was not the cause, I undo it. :-)